website/docs/d/nsxt_edgegateway_l2_vpn_tunnel.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_l2_vpn_tunnel"
+sidebar_current: "docs-vcd-datasource-nsxt-edgegateway-l2-vpn-tunnel"
```

Comment:
Adding these two new pages to `vcd.erb` is missing.

Reply:
True, added